sentry-android-core/src/main/java/io/sentry/android/core/internal/gestures/AndroidViewGestureTargetLocator.java

```diff
@@ -0,0 +1,80 @@
+package io.sentry.android.core.internal.gestures;
+
+import android.view.View;
+import android.widget.AbsListView;
+import android.widget.ScrollView;
+import androidx.core.view.ScrollingView;
+import io.sentry.internal.gestures.GestureTargetLocator;
+import io.sentry.internal.gestures.UiElement;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@ApiStatus.Internal
+public final class AndroidViewGestureTargetLocator implements GestureTargetLocator {
+
+  private final boolean isAndroidXAvailable;
+  private final int[] coordinates = new int[2];
+
+  public AndroidViewGestureTargetLocator(final boolean isAndroidXAvailable) {
+    this.isAndroidXAvailable = isAndroidXAvailable;
+  }
+
+  static boolean isViewTappable(final @NotNull View view) {
+    return view.isClickable() && view.getVisibility() == View.VISIBLE;
+  }
+
+  static boolean isViewScrollable(final @NotNull View view, final boolean isAndroidXAvailable) {
+    return (isJetpackScrollingView(view, isAndroidXAvailable)
+            || AbsListView.class.isAssignableFrom(view.getClass())
+            || ScrollView.class.isAssignableFrom(view.getClass()))
+        && view.getVisibility() == View.VISIBLE;
+  }
+
+  private static boolean isJetpackScrollingView(
+      final @NotNull View view, final boolean isAndroidXAvailable) {
+    if (!isAndroidXAvailable) {
+      return false;
+    }
+    return ScrollingView.class.isAssignableFrom(view.getClass());
+  }
+
+  static boolean touchWithinBounds(
```

Comment:
`l`: do we still need to pass the `coords` parameter or can just use `coordinates` directly since it's a class member now?